instrumentation/aws-sdk/aws-sdk-2.2/testing/build.gradle.kts

```diff
@@ -8,14 +8,15 @@ dependencies {
   api("software.amazon.awssdk:apache-client:2.2.0")
   // older versions don't play nice with armeria http server
   api("software.amazon.awssdk:netty-nio-client:2.11.0")
-  // When adding libraries, make sure to also add to the library/javaagent build files
-  // to ensure they are bumped to the latest version under testLatestDeps
-  api("software.amazon.awssdk:dynamodb:2.2.0")
-  api("software.amazon.awssdk:ec2:2.2.0")
-  api("software.amazon.awssdk:kinesis:2.2.0")
-  api("software.amazon.awssdk:rds:2.2.0")
-  api("software.amazon.awssdk:s3:2.2.0")
-  api("software.amazon.awssdk:sqs:2.2.0")
+
+  // compileOnly because we never want to pin the low version implicitly; need to add dependencies
```

Comment:
This was required to allow using this from the testCoreOnly test suite without also dragging in SQS. It required a slew of follow-up changes, but I think the end result is a bit cleaner, and also aligned with what the v1.1 SDK does.